product_replenishment_cost/views/product_view.xml

```diff
@@ -6,7 +6,7 @@
         <field name="inherit_id" ref="product.product_normal_form_view" />
         <field name="arch" type="xml">
             <xpath expr="//field[@name='standard_price']/ancestor::*[1]" position="after"> 
-                <field name="replenishment_cost"/> 
+                <field name="replenishment_cost"/>
             </xpath>
         </field>
     </record>
```

Comment:
I guess that's a typo since `product_standard_margin` doesn't depend of `product_replenishment_cost`

Reply:
Fixed ! Thanks !